### DISCUSSION TITLE
Add support for reference image

### DISCUSSION THREAD

**Discussion by fian46** [Orama-Interactive/Pixelorama#565] [Ideas]
Can i request feature for reference image like in krita ?

**Comment by OverloadedOrama** [comment]
I've been planning to implement a reference image feature, and it's most likely going to be implemented in v0.7!

EDIT: Feature pushed to v0.9.
EDIT 2: I'm not sure when this will get implemented, as other features have higher priority right now.

**Comment by OverloadedOrama** [comment]
Implemented by #771.

**Reply by fian46** [reply]
is this already in github release ? what version can i use this feature ?

**Reply by Variable-ind** [reply]
Hy 👋, It's in the nightly builds you can get it here https://www.oramainteractive.com/Pixelorama-Docs/user_manual/installation#early-access---nightly-versions
Or go directly to download section https://nightly.link/Orama-Interactive/Pixelorama/workflows/dev-desktop-builds/master
